tests/sync/test_syncing.py

```diff
@@ -91,3 +136,151 @@ def test_complex_sync_all(self, tmpdir: py.path.local):
             pathlib.Path(f'{tmpdir}/syncs/sync-2/group4-file1.txt'),
             pathlib.Path(f'{tmpdir}/syncs/sync-2/group4-file2.txt'),
         ]
+
+
+class TestConfigError:
+    def test_valid_configuration(self, tmpdir: py.path.local):
```

Comment:
Please add a blank line above this (kind of surprised flake8/pylint don't show a warning for this!)

Reply:
Ok, so always a new line after a class-statement?